nilearn/datasets/tests/test_func.py

```diff
@@ -362,3 +363,48 @@ def test_fetch_mixed_gambles():
         assert_equal(mgambles["zmaps"][0], os.path.join(datasetdir, "zmaps",
                                                         "sub001_zmaps.nii.gz"))
         assert_equal(len(mgambles["zmaps"]), n_subjects)
+
+
+def test_wrong_inputs_of_dimensionality_timeseriesmethods_matrices():
+
+    message = "Invalid %s name"
+
+    dimensionality = ['a10', 'd15', 'd30']
+    assert_raises_regex(ValueError,
+                        message % 'dimensionality',
```

Comment:
It would be great to test that the invalid value is part of the error message. Also I would really make it obvious which one it is for people not familiar with this fetcher, i.e. call it 'invalid_input' rather than 'a10', 'multiple_sptial_regression', etc ...